### PR TITLE
Fix naming of plates

### PR DESCRIPTION
**Pull request by withrocks**

### CODE REVIEW

File: clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class Extension(GeneralExtension):
 
 
     Creates two containers with samples and controls in Clarity:
-        COVID_PREXT_<timestamp> 
+        COVID_PREXT_<timestamp>
```
Comment (GitEdvard) [inline clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py:28]: This comment is no longer according to implementation

File: clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py [hunk 2]
```diff
@@ -101,7 +103,9 @@ def create_in_mem_container(
         return container
 
     def execute(self):
-        timestamp = self.context.start.strftime("%Y%m%dT%H%M%S")
+        start = self.context.start
```
Comment (GitEdvard) [inline clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py:106]: What if a lims-user decides to open up the create samples step, and then never exit it? (She doesn't have to exit it between subsequent logins) Is the "start" timestamp going to be the same all the time?

Comment (withrocks) [reply clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py:106]: No. This is the start of the execution of the script.

File: clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py (hunk 1 above)
Comment (withrocks) [reply clarity-ext-scripts/clarity_ext_scripts/covid/import_samples.py:28]: Good call!

**Pull request merged**